Configure WASD keys for panning

rockokw opened the pull request:
Closes #70

cb299792458 commented on frontend/src/components/GameDisplay.jsx:
```diff
@@ -17,8 +17,21 @@ export default function GameDisplay({ game, assets, selectedTower }) {
     const { gameMap, cameraTarget } = game;
     const { width, depth, height } = gameMap;
 
+    const [ready, setReady] = useState(false);
```
Is `ready` necessary? What happens without this check?

rockokw replied:
I had run into an issue where the keys would randomly stop working after a soft/hard refresh. I haven't quite figured out the cause yet (one idea is that it could be related to dev mode or hot reloading). The `ready` variable helps to make it work consistently.

If I end up keeping it there, I'll add a comment explaining.

cb299792458 reviewed (commented):
It's working nicely, but could we try it with W/S being forward/back instead of up/down? I feel like up/down isn't as useful in the game currently, and they could be mapped to Q/E/R/C/etc instead. Something like this https://codesandbox.io/s/r3f-orbit-controls-un2oh feels a little more natural to me (uses arrow keys, but for four horizontal directions, not vertical) I also think a button to reset the camera would be nice, in case you pan too far somehow and get lost.

rockokw commented:
> It's working nicely, but could we try it with W/S being forward/back instead of up/down? I feel like up/down isn't as useful in the game currently, and they could be mapped to Q/E/R/C/etc instead. Something like this https://codesandbox.io/s/r3f-orbit-controls-un2oh feels a little more natural to me (uses arrow keys, but for four horizontal directions, not vertical) I also think a button to reset the camera would be nice, in case you pan too far somehow and get lost.

Ok, fixed. It was just relative to the camera before, as opposed to the ground plane. I also added a comment about the `ready` state variable.

Regarding reset, try the 'c' key. I set that one up in a previous PR. At some point I'll likely also bound the camera movement so you can't get too far from the map. Some limits on angle and zoom are already in place.
